packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -547,6 +547,18 @@ describe('function', () => {
         Annotations.fromStack(stack).hasNoWarning('/Default/MyLambda/$LATEST', Match.stringLikeRegexp(warningMessage));
       });
 
+      test('function.addAlias', () => {
```

Comment:
If I'm understanding correctly, the issue with `fn.currentVersion.addAlias()` is that the Alias will be replaced on every function update. I think we should test that `fn.addAlias()` solves the issue. This test doesn't really tell me anything.

To be honest, I'm not clear how `function.addAlias()` changes this behavior. To me, it seems like both `fn.addAlias()` and `fn.currentVersion.addAlias()` results in the same thing: `return addAlias(this, <current-version>, aliasName, options)`.

Reply:
Yes, but the `this` is different! In one case it's the `Function` and the other it's the `Version`.

But I think I was mistaken anyway: I thought the `Alias` would be replaced if the `Version` object changed... but I'm now pretty convinced it doesn't.

Reply:
So this is just convenience.